superset/sql_validators/presto_db.py

```diff
@@ -157,7 +157,7 @@ def validate(
         annotations: List[SQLValidationAnnotation] = []
         with closing(engine.raw_connection()) as conn:
             with closing(conn.cursor()) as cursor:
-                for statement in parsed_query.get_statements():
+                for statement in statements:
```

Comment:
this could be in a separate PR right? Might be better to pull it out since it's unrelated to this set of work

Reply:
Yeah, I was looking at this method and saw that `statements` was already computed. I'll write a separate PR.